far/farlang.templ.m4

```diff
@@ -1,4 +1,4 @@
-#
+﻿#
```

Comment:
Did something happen to the BOM here, did it dis-/appear? Won't it confuse the build pipeline tools?

Reply:
IMHO if it's UTF-8 and BOM does not break the build, we should leave it.